pkg/app/pagewriter/error_page_test.go

```diff
@@ -28,41 +29,75 @@ var _ = Describe("Error Page Writer", func() {
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      403,
+				RedirectURL: "/redirect",
+				RequestID:   requestID,
+				AppError:    "Access Denied",
+			})
 
 			body, err := ioutil.ReadAll(recorder.Result().Body)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(string(body)).To(Equal("Forbidden You do not have permission to access this resource. /prefix/ 403 /redirect Custom Footer Text v0.0.0-test"))
+			Expect(string(body)).To(Equal("Forbidden You do not have permission to access this resource. /prefix/ 403 /redirect 11111111-2222-4333-8444-555555555555 Custom Footer Text v0.0.0-test"))
 		})
 
 		It("With a different code, uses the stock message for the correct code", func() {
 			recorder := httptest.NewRecorder()
-			errorPage.WriteErrorPage(recorder, 500, "/redirect", "Access Denied")
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      500,
+				RedirectURL: "/redirect",
+				RequestID:   requestID,
+				AppError:    "Access Denied",
+			})
 
 			body, err := ioutil.ReadAll(recorder.Result().Body)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(string(body)).To(Equal("Internal Server Error Oops! Something went wrong. For more information contact your server administrator. /prefix/ 500 /redirect Custom Footer Text v0.0.0-test"))
+			Expect(string(body)).To(Equal("Internal Server Error Oops! Something went wrong. For more information contact your server administrator. /prefix/ 500 /redirect 11111111-2222-4333-8444-555555555555 Custom Footer Text v0.0.0-test"))
 		})
 
 		It("With a message override, uses the message", func() {
 			recorder := httptest.NewRecorder()
-			errorPage.WriteErrorPage(recorder, 403, "/redirect", "Access Denied", "An extra message: %s", "with more context.")
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      403,
+				RedirectURL: "/redirect",
+				RequestID:   requestID,
+				AppError:    "Access Denied",
+				Messages: []interface{}{
+					"An extra message: %s",
+					"with more context.",
+				},
+			})
+
+			body, err := ioutil.ReadAll(recorder.Result().Body)
+			Expect(err).ToNot(HaveOccurred())
+			Expect(string(body)).To(Equal("Forbidden An extra message: with more context. /prefix/ 403 /redirect 11111111-2222-4333-8444-555555555555 Custom Footer Text v0.0.0-test"))
+		})
+
+		It("Sanitizes malicious user input", func() {
+			recorder := httptest.NewRecorder()
+			errorPage.WriteErrorPage(recorder, ErrorPageOpts{
+				Status:      403,
+				RedirectURL: "/redirect",
+				RequestID:   "<script>alert(1)</script>",
+				AppError:    "Access Denied",
+			})
 
 			body, err := ioutil.ReadAll(recorder.Result().Body)
 			Expect(err).ToNot(HaveOccurred())
-			Expect(string(body)).To(Equal("Forbidden An extra message: with more context. /prefix/ 403 /redirect Custom Footer Text v0.0.0-test"))
+			Expect(string(body)).To(Equal("Forbidden You do not have permission to access this resource. /prefix/ 403 /redirect  Custom Footer Text v0.0.0-test"))
```

Comment:
So that sanitizer just removed it all? Is that the intention here?

Reply:
Yeah -- there's a double space in the output where an empty {{.RequestID}} resides.

Reply:
I could use html escape instead of a sanitizer? 🤔

Reply:
HTML escape would probably be easier to understand wouldn't it? Then you at least see that there was something there rather than an empty string? Thinking from a UX perspective here